Formula/php.rb

```diff
@@ -0,0 +1,344 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net"
+  url "https://github.com/php/php-src/archive/php-7.1.8.tar.gz"
+  sha256 "f3be4262203fa9db3d126832a1632e9a80e52e313f4230218c392d6a9178c368"
+
+  # javian: not sure about the origin of this so I'll keep it commented for now. Could it have something to do with building extensions ?
+  # javian: This has been in the formula for 8+ years but have found no proof that it is needed anymore
+  # So PHP extensions don't report missing symbols
+  # skip_clean "bin", "sbin"
+
+  # TODO
+  # Opcache default config file is included in extension formula, is it needed ?
+  # ldap extension: I have a vague recollection that this caused some issues that were resolved with exluding it. needs to be checked.
+  # Need to add tests for all supported SAPIs
+
+  option "with-imap-uw", "Build PHP IMAP extension"
+  option "with-thread-safety", "Build with thread safety"
+
+  # javian: Not yet checked these options
+  option "with-pdo-oci", "Include Oracle databases (requries ORACLE_HOME to be set)"
+  option "without-ldap", "Build without LDAP support"
+  option "without-unixodbc", "Build without unixODBC support"
+
+  depends_on "autoconf" => :build
+  depends_on "bison" => :build
+  depends_on "re2c" => :build
+  depends_on "imap-uw" => :optional
+  depends_on "libtool" => :run # javian: mcrypt requirement
+  depends_on "aspell"
+  depends_on "argon2"
+  depends_on "curl" if MacOS.version < :lion
+  depends_on "enchant"
+  depends_on "freetds"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "homebrew/apache/httpd24"
+  depends_on "icu4c"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libpq"
+  depends_on "mcrypt"
+  depends_on "net-snmp"
```

Comment:
The macOS version should suffice, and it's not currently explicitly referenced in the configure options.

Reply:
The system `net-snmp` on newer versions of macOS is linked to LibreSSL as well, so you'd end up with a mixed crypto tree. Using the dep is a saner behaviour here.